source/quickstart/windows.rst

```diff
@@ -6,31 +6,115 @@ Quickstart for Windows
 
 .. default-domain:: minio
 
-This page deploys MinIO on Windows for use with early development
-and evaluation environments.
+.. |OS| replace:: Windows
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+Use the procedures on this page to deploy MinIO in :guilabel:`Standalone Mode` on a single local |OS| harddrive as a single MinIO server process (what we call "filesystem mode") with a :ref:`subset of S3 features<minio-installation-comparison>`.
```

Comment:
```suggestion
Use the procedures on this page to deploy MinIO in :guilabel:`Standalone Mode` on a single local |OS| harddrive as a single MinIO server process (what we call "filesystem mode") with a :ref:`subset of S3 features <minio-installation-comparison>`.
```

Reply:
The space might not be necessary from a parser point of view but I'd still put it in there - makes find-replace a little easier too.